scripts/build-toolchains.sh

```diff
@@ -34,6 +35,7 @@ die() {
 
 TOOLCHAIN="riscv-tools"
 EC2FASTINSTALL="false"
+IGNOREQEMU="false"
```

Comment:
Nit: avoid the negation, and just have a `INSTALLQEMU` variable.

Reply:
I'm following what the `EC2FASTINSTALL` did... let me double-check that there wasn't a specific reason to do that. I'm not opposed to just checking if the variable exists (assuming it doesn't break things).

Reply:
Hmm. This looks like this was just coding preference. Ill change it.